### PR TITLE
Enable embedded symbols

### PR DESCRIPTION
**Pull request by pakrym-stripe**
Makes the debugging simpler. (https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/nuget#symbol-packages)

Similar to https://github.com/stripe/stripe-dotnet/pull/2427 but doesn't require an additional package reference.

### CODE REVIEW

File: src/StripeTests/StripeTests.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;net6.0;netcoreapp3.1;netcoreapp2.1;net461</TargetFrameworks>
```
Comment (dcr-stripe) [inline src/StripeTests/StripeTests.csproj:4]: We're dropping netcoreapp2.1 - here and below - is that expected?

File: src/Stripe.net/Stripe.net.csproj [hunk 2]
```diff
@@ -17,6 +17,7 @@
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <IncludeSymbols>true</IncludeSymbols>
+    <DebugType>embedded</DebugType>
```
Comment (dcr-stripe) [inline src/Stripe.net/Stripe.net.csproj:20]: Does this get included in our release artifact/grow its size?

**Review by dcr-stripe — COMMENTED** [review]
PTAL @pakrym-stripe

File: src/StripeTests/StripeTests.csproj (hunk 1 above)
Comment (pakrym-stripe) [reply src/StripeTests/StripeTests.csproj]: Added it back here. Below is fine. We don't need a separate package list for every target framework.

File: src/Stripe.net/Stripe.net.csproj (hunk 2 above)
Comment (pakrym-stripe) [reply src/Stripe.net/Stripe.net.csproj:20]: Yes, adds 100kb to a 3.9mb package.

**Pull request merged**